apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/RateLimiter.cs

```diff
@@ -27,7 +27,7 @@ internal sealed class RateLimiter
         private static object _instanceMutex = new object();
 
         /// <summary>The single rate limiter instance.</summary>
-        private static RateLimiter _instance;
+        internal static RateLimiter _instance;
```

Comment:
I would probably have a `Reset()` static method or something similar rather than make this internal.